docs/mast/mast.rst

```diff
@@ -458,7 +458,8 @@ The Catalogs class provides access to a subset of the astronomical catalogs stor
 - The TESS Input Catalog (TIC)
 - The TESS Candidate Target List (CTL)
 - The Disk Detective Catalog
-- PanSTARRS (DR1, DR2)
+- The PanSTARRS Catalog (DR1 and DR2)
+- The PLATO Catalog (DR1)
```

Comment:
Is this the asPIC catalog or something else? Either case, calling it PLATO catalog DR1 is confusing given PLATO hasn't yet been launched.

Reply:
Yes, this is referring to asPIC, and I just confirmed w a team member. Documentation is now updated to reflect this